test/res.set.js

```diff
@@ -2,19 +2,19 @@
 var express = require('..');
 var request = require('supertest');
 
-describe('res', function(){
-  describe('.set(field, value)', function(){
-    it('should set the response header field', function(done){
+describe('res', function () {
+  describe('.set(field, value)', function () {
+    it('should set the response header field', function (done) {
       var app = express();
 
-      app.use(function(req, res){
+      app.use(function (req, res) {
         res.set('Content-Type', 'text/x-foo; charset=utf-8').end();
       });
 
       request(app)
-      .get('/')
-      .expect('Content-Type', 'text/x-foo; charset=utf-8')
-      .end(done);
+        .get('/')
```

Comment:
Typically we do not indent these lines, keeping the `.` at the same indent as the function call. Can you revert all the re-formatting of the file that wasn't related to your changes?